.gitignore

```diff
@@ -17,3 +17,5 @@ vgcore.*
 /dunst.systemd.service
 /org.knopwob.dunst.service
 /test/test
+
+.idea/
```

Comment:
I use CLion from IntelliJ. It uses this directory to store project-specific settings :)

Reply:
I use CLion from IntelliJ. It uses this directory to store project-specific settings :)

Reply:
Might be better to put `.idea` into your global `~/.config/git/ignore` then.

Reply:
I did not push this commit as it was purely cosmetic and I was sure there will be something else :laughing: And I was right, as I changed the test/test target, it inevitably broke the clang build. At first I did not really pay attention but I went through it as I saw you did something and all builds have passed. My bad.

It will be gone momentarily as I fix the clang build :slightly_smiling_face: